nimterop/globals.nim

```diff
@@ -1,4 +1,4 @@
-import sequtils, sets, tables
+import segfaults, sequtils, sets, tables
```

Comment:
add: pending https://github.com/genotrance/nimterop/issues/98 ; this import has implications (and its own issues)

Reply:
Can you explain? Seems nice to get a stack trace when things crash.

Reply:
so it's indeed nice when debugging, but i don't think it should be enabled by default as it resets signal handler (eg which a user could've set etc)